### DISCUSSION TITLE
Set a default authentication method on MinIO

### DISCUSSION THREAD

**Discussion by olivier-jobert** [minio/minio#16928] [Q&A]
Hello,

I'm using the Helm Chart Vanilla to deploy MinIO which is configured with Keycloak as identity provider (OIDC).

Is it possible to automatically redirect users to the SSO login page instead of letting them choose which authentication they want to use?

![image](https://user-images.githubusercontent.com/20954716/228866970-9b08eab2-d422-4e0c-b1a5-3676f99cbe66.png)

I took a look on values.yaml, documentation and MinIO Console source code, but did not found a simple way to set a default authentication method (*in my case I want to use SSO by default*).

Is this out-of-the-box possible?

Thanks

**Comment by harshavardhana** [comment]
@bexsoft @dvaldivia ^^

**Comment by peter-iglaev** [comment]
hey, team.
same issue.

any updates?
we need to decide if we should use previous minio version or wait for fix for new?

**Comment by bexsoft** [comment]
Hi @peter-iglaev!

We made some changes and improvements to Login page, now SSO option will be seen as default and Login forms will be available inside the `Other Authentication methods` button. Please refer to https://github.com/minio/console/pull/2807.

This change is available in the latest versions of MinIO.

**Reply by peter-iglaev** [reply]
hey, @bexsoft.
Cool! 
Thank you a lot!
